hazelcast/src/test/resources/test-hazelcast-real-jcache.xml

```diff
@@ -0,0 +1,80 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Is this `test-hazelcast-real-jcache.xml` used only by this test ? If it is, it may better to clean unnecessary configurations from this xml configuration to make it clean.